docs/website/blog/2024-03-13-smart-dashboarding-tools.md

```diff
@@ -0,0 +1,141 @@
+
+## Evaluating Sage AI
+
+Before diving in, it's important to note that ThoughtSpot can connect to [dbt's semantic layer](https://dlthub.com/docs/blog/dlt-dbt-semantic-layer), which helps contextualize the questions asked - for example, by making clear what a certain success metric might mean. However, we haven't set up any semantic definitions or pre-built dashboards for this data. Our analysis will solely rely on raw data modeled by `dlt`. This approach might limit our ability to fully showcase ThoughtSpot's capabilities and the potential of combining AI in analytics with semantic modeling. Nonetheless, our focus here is on evaluating Sage AI's inference capabilities with raw data, across all categories except for prescriptive analytics, which leans more towards strategic business decision-making.
+
+The lack of semantic definitions and other dashboards meant that for each question asked to Sage, we had to specify exactly which table it should look into, to find a result. For example:
+
+![example question](https://storage.googleapis.com/dlt-blog-images/smart-dashboarding-question-example.png)
+
+Let’s begin asking questions and see how Sage answers! The framing of each question is exactly as it was asked to Sage.
+
+### Descriptive Analytics
+
+1. How many companies do we have? ✅
+2. How many companies do we serve? ✅
+3. How many deals do we have by month? ✅
+4. Deals broken by industry, shown as percentages ❌
+5. Revenue of deal by stage ❌ Proportion of each deal stage’s revenue ❌
+6. What percentage of companies are in the computer software industry? ✅
+7. Show revenue by month ✅ **Worked even though the revenue column is named “amount” - it could infer!**
+8. How many deals are signed by each company? **✅**
+
+    This was by far the most astounding result! The underlying model is `companies__deals`, and it contains the unnested information for what deals belong to which company. The schema looks like this:
+   ![schema](https://storage.googleapis.com/dlt-blog-images/smart-dashboarding-data-model-2.png)
+   a. 1. Unnested means that there is a parent table, which here is `companies`, and a child table, which here is `deals`. A company can have several deals.
+   b. The `_dlt_parent_id` then refers to a company id, saved in the companies table. This is a `dlt` assigned primary key to a company. Whereas, the `value` field is a HubSpot defined primary key to a company. Both saved as foreign keys in this table.
+   c. The `deals_id` is also therefore present in the `deals` table.
+   d. Whereas, `_dlt_list_idx` is another column to keep track of rows during the process of unnesting - courtesy of `dlt`.
+   Perhaps given the naming convention of the table; companies__deals, and the word parent in the columns, Sage was able to infer the connection between the two. Here is the result:
```

Comment:
If you want to break the line you should add extra empty line here, otherwise it looks like solid text
![image](https://github.com/dlt-hub/dlt/assets/20367975/3d255bae-d6d2-43d5-901a-eaa32c08a938)